Documentation/contributing/development/dev_setup.rst

```diff
@@ -90,16 +107,25 @@ You can find the setup for a `kind <https://kind.sigs.k8s.io/>`_ environment in
 VirtualBox on Linux, but does require `Docker for Mac
 <https://docs.docker.com/desktop/install/mac-install/>`_ for Mac OS.
 
-Makefile targets automate the task of spinning up an environment and building
-Cilium images:
+Makefile targets automate the task of spinning up an environment:
 
 * ``make kind``: Creates a kind cluster based on the configuration passed in.
-  For more information, see _`Configuration for clusters`.
+  For more information, see `configurations_for_clusters`.
+* ``make kind-down``: Tears down and deletes the cluster.
+
+Depending on your environment you can build Cilium by using the following
+makefile targets:
+
+For Linux and Mac OS
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
I realise our theme does not make a visual distinction between level 3 and level 4 headings :grimacing:

Reply:
It doesn't? It looked to me that the level 4 had a smaller font size.

Reply:
Alright, 18.75px vs. 17.25px. The difference is thin but it exists, I had missed it. Not something for this PR anyway.